NetCore/LoadResizeSave.cs

```diff
@@ -303,13 +306,33 @@ internal static void SkiaBitmapLoadResizeSave(string path, int size, string outp
                     }
 
                     using (var image = SKImage.FromBitmap(resized))
-                    using (var output = File.OpenWrite(OutputPath(path, outputDirectory, SkiaSharpBitmap)))
+                    using (var output = File.OpenWrite(OutputPath(input, SkiaSharpBitmap)))
                     {
                         image.Encode(SKEncodedImageFormat.Jpeg, Quality)
                              .SaveTo(output);
                     }
                 }
             }
         }
+
+        [Benchmark(Description = "NetVips Load, Resize, Save"/*,
+            OperationsPerInvoke = ImagesCount*/)]
+        public void NetVipsBenchmark()
+        {
+            foreach (string image in Images)
+            {
+                NetVipsResize(image);
+            }
+        }
+
+        public void NetVipsResize(string input)
+        {
+            // Thumbnail to fit a 150x150 square
+            using (var thumb = NetVipsImage.Thumbnail(input, ThumbnailSize, ThumbnailSize))
+            {
+                // Save the results
+                thumb.Jpegsave(OutputPath(input, NetVips), q: Quality, strip: true);
```

Comment:
Is this running the same operation as other tests?

Reply:
Do you mean the [thumbnail part](https://libvips.github.io/libvips/API/current/libvips-resample.html#vips-thumbnail) or the [jpegsave part](https://libvips.github.io/libvips/API/current/VipsForeignSave.html#vips-jpegsave)?

Reply:
`Thumbnail` vs `Reduce` I mean. A very quick read of the docs it seems `Reduce` is using a simple transform to do the resizing. 

Perhaps `Resize` as you have above would be best for both since `Thumbnail` likely has jpeg specific optimizations (I'm guessing here) and while the test images are jpeg, it's probably best to compare methods that work in a similar fashion across multiple image formats.

Reply:
Hello, `reduce` does adaptive convolution with a choice of linear / cubic / lanczos etc. kernel.

`thumbnail` does shrink-on-load as appropriate for the filetype, followed by a box filter for any huge reductions, then `reduce` for the final 200%+ of the resize.

Reply:
Thanks @jcupitt I wasn't sure whether `Reduce` performed other operations (alpha premultiplication etc) like many resampling algorithms and thanks for confirming `Thumbnail` behaviour. 

Very interesting idea combining box with other methods btw. I like it!

Reply:
Yes, it does premul/unpremul, though only for images with alpha channels. It'll transform to and from a reduction space too, so you can shrink in linear light, or shrink LCh images. Eight bit images have a path with run-time vector code generation for some of the steps.

Reply:
I switched from `Reduce` to `Resize` within PR #17.